etc/queue_topology.xml

```diff
@@ -38,4 +38,7 @@
     <exchange name="magento-db" type="topic" connection="db">
         <binding id="updateBinding" topic="nosto_product_sync.update" destinationType="queue" destination="nosto_product_sync.update"/>
     </exchange>
+    <exchange name="magento-db" type="topic" connection="db">
```

Comment:
This will throw an exception. Must be 
```suggestion
    <exchange name="magento-db" type="topic" connection="db">
        <binding id="updateBinding" topic="nosto_product_sync.update" destinationType="queue" destination="nosto_product_sync.update"/>
        <binding id="deleteBinding" topic="nosto_product_sync.delete" destinationType="queue" destination="nosto_product_sync.delete"/>
    </exchange>
```